src/Control/Control.js

```diff
@@ -1516,7 +1516,7 @@ var Control = module.exports = kind(
 		var was = this.content;
 		this.content = content;
 
-		if (was != content) this.notify('content', was, content);
+		if (was !== content) this.notify('content', was, content);
```

Comment:
I think Cole's argument against this was that we should not be setting non-string values into content so we would be hiding developer error.